hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -183,6 +183,22 @@ class ActuatorInterface : public rclcpp_lifecycle::node_interfaces::LifecycleNod
         info_.thread_priority);
       async_handler_->start_thread();
     }
+
+    if (auto locked_executor = params.executor.lock())
+    {
+      std::string node_name = params.hardware_info.name;
+      std::replace(node_name.begin(), node_name.end(), '/', '_');
+      hardware_component_node_ = std::make_shared<rclcpp::Node>(node_name);
+      locked_executor->add_node(hardware_component_node_->get_node_base_interface());
+    }
+    else
+    {
+      RCLCPP_ERROR(
+        params.logger,
+        "Executor is not available during hardware component initialization for '%s'.",
+        params.hardware_info.name.c_str());
```

Comment:
```suggestion
      RCLCPP_WARN(
        params.logger,
        "Executor is not available during hardware component initialization for '%s'. Skipping node creation!",
        params.hardware_info.name.c_str());
```

Should it be warning or error?

Reply:
Yep warning